appveyor.yml

```diff
@@ -28,3 +28,19 @@ after_build:
 
 build_script:
 - cmd: build-vs2017.cmd
+
+artifacts:
+- path: build\*.nupkg
+  name: core
+
+deploy:
+- provider: NuGet
+  api_key:
+    secure: Please insert key here
```

Comment:
> It appears from the AppVeyor docs that we could define this with a secure environment variable rather than having to encrypt the API token and store it here, I'd much prefer that if possible.

Can you check this out and reconsider: https://www.appveyor.com/docs/build-configuration/#secure-variables